src/SFA.DAS.EmployerApprenticeshipsService.Domain/Data/Entities/Account/Primitives/PayeAccountRef.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EAS.Domain.Data.Entities.Account.Primitives
+{
```

Comment:
This should not be committed at the moment as it looks like its been removed from the project.

Reply:
Deleted from disc